cmake/Modules/FindFmt.cmake

```diff
@@ -0,0 +1,15 @@
+# - Try to find Fmt
+# Once done, this will define
+#
+#  Fmt_FOUND - system has fmt
+#  Fmt_INCLUDE_DIRS - the fmt include directories
+#  Fmt_LIBRARIES - link these to use fmt
```

Comment:
I still dislike detecting dependencies like this without using the canonical way of doing it in modern cmake.The imported target should be fmt::fmt, and all this file be one line...
As I seem to be the only one with this opinion, I will disagree and commit.

Reply:
How would you like to see this find module?

Reply:
Resolved as discussed on discord. Opened https://github.com/performous/performous/issues/767 added to resolve the other issues regarding dependencies